doc/releases/release-notes-3.7.rst

```diff
@@ -663,6 +663,12 @@ Drivers and Sensors
 
 * Crypto
 
+* DAC
+
+  * Added support for NXP RW SOC series DAC.
+  * Added support for Analog Devices AD5691 / AD5692 / AD5693 DACs.
+  * Added support for Texas Instruments DACx0501 series DACs.
```

Comment:
Actually could you add links to the dt compatibles?

Reply:
Done.